### PR TITLE
Add Archway <> JUNO IBC

### PR DESCRIPTION
**Pull request by Rizbe**

### CODE REVIEW

File: _IBC/archway-juno.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../ibc_data_schema.json",
+  "chain_1": {
+    "chain_name": "archway-1",
```
Comment (JeremyParish69) [inline _IBC/archway-juno.json:4]: there chain ids need to be changed to chain names. You can just remove the '-1'

**Comment by Rizbe** [comment]
@JeremyParish69 
Noticed here looking at JUNO<>OSMO: https://github.com/cosmos/chain-registry/blob/614bb87ba1daff10b6e0e6de1fcb56d8b57b9c2d/_IBC/juno-osmosis.json#L34


Not able to IBC CW20 token (GLTO) using our relayer. is it because we need something like the above?

**Comment by Rizbe** [comment]
> > @JeremyParish69 Noticed here looking at JUNO<>OSMO:
> > https://github.com/cosmos/chain-registry/blob/614bb87ba1daff10b6e0e6de1fcb56d8b57b9c2d/_IBC/juno-osmosis.json#L34
> > 
> > Not able to IBC CW20 token (GLTO) using our relayer. is it because we need something like the above?
> 
> CW20 tokens need a special CW20-to-ICS20 contract on the originating chain and another channel to be able to IBC transfer them; that's what that port is--the contract

Any doc around this I can follow to get this going <3

**Comment by JeremyParish69** [comment]
Might start here, @Rizbe ?
https://github.com/CosmWasm/cw-plus/tree/main/contracts/cw20-ics20

**Pull request merged**